src/javascript/app_2/App/Components/Elements/Inkbar/inkbar.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React     from 'react';
+
+export default function Inkbar({ left, width }) {
```

Comment:
How about using function expression like other places?
```js
export default InkBar = ({ left, width }) => {
```
And also renaming to `InkBar` everywhere.